sphinx/ext/autodoc/__init__.py

```diff
@@ -2845,24 +2778,6 @@ def autodoc_attrgetter(app: Sphinx, obj: Any, name: str, *defargs: Any) -> Any:
     return safe_getattr(obj, name, *defargs)
 
 
-def migrate_autodoc_member_order(app: Sphinx, config: Config) -> None:
-    if config.autodoc_member_order == 'alphabetic':
-        # RemovedInSphinx50Warning
-        logger.warning(__('autodoc_member_order now accepts "alphabetical" '
-                          'instead of "alphabetic". Please update your setting.'))
-        config.autodoc_member_order = 'alphabetical'  # type: ignore
```

Comment:
`"alphabetic"` should also be removed from the "add_config_value" call.

Reply:
I'll do this in another PR after the merge.

Reply:
Done

Reply:
Wow!